hvantk/datasets/cptac_phospho_datasets.py

```
@@ -301,17 +301,27 @@ def download(self, output_dir: str, overwrite: bool = False) -> Dict[str, str]:
         logger.info("Loading CPTAC %s dataset...", ct)
         ds = _load_cptac_dataset(ct)
 
+        def _get_phospho(tissue_type="both", source="umich"):
+            """Try *source* first; fall back to default if unavailable."""
+            try:
+                return ds.get_phosphoproteomics(source=source, tissue_type=tissue_type)
+            except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)